config/config.go

```diff
@@ -405,6 +405,11 @@ type RPCConfig struct {
 
 	// pprof listen address (https://golang.org/pkg/net/http/pprof)
 	PprofListenAddress string `mapstructure:"pprof_laddr"`
+
+	// Set false to ensure ABCI responses are persisted.
+	// ABCI responses are required for /BlockResults RPC queries, and
+	// to reindex events in the command-line tool.
+	DiscardABCIResponses bool `mapstructure:"persist-abci-responses"`
```

Comment:
```suggestion
	// Set false to ensure ABCI responses are persisted.
	// ABCI responses are required for /BlockResults RPC queries, and
	// to reindex events in the command-line tool.
	DiscardABCIResponses bool `mapstructure:"discard_abci_responses"`
```

The keys changed from `this_style` to `this-style` between releases. I'm also realizing that the config key is `persist-abci-responses` but the field name is `DiscardABCIResponses` which have opposite meanings. I think we should change the key to match the field name.

Reply:
Yeah this completely makes sense, Ill make those updates